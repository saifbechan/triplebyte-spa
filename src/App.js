import React from 'react';

import Bytes from './components/bytes/bytes.component';
import Octal from './components/octal/octal.component';

import { AppContainer } from './App.styles';

const App = () => (
  <AppContainer>
    <Bytes />
    <Octal />
  </AppContainer>
);

export default App;
