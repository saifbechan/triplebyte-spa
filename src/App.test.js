import React from 'react';
import { render } from '@testing-library/react';

import App from './App';

jest.mock('./components/bytes/bytes.component', () => 'div');
jest.mock('./components/octal/octal.component', () => 'div');

it('should render correctly', () => {
  const { asFragment } = render(<App />);
  expect(asFragment()).toMatchSnapshot();
});
