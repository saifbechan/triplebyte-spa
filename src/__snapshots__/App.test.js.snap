// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<DocumentFragment>
  <div
    class="sc-AykKC dkqCea"
  >
    <div />
    <div />
  </div>
</DocumentFragment>
`;
